pkg/oc/cli/admin/migrate/scc/scc_to_psp.go

```diff
@@ -267,12 +266,12 @@ func extractSeccompProfiles(scc *securityapi.SecurityContextConstraints, annotat
 	}
 }
 
-// Sysctl support is implemented as an annotation in both OpenShift and Kubernetes.
-func extractSysctls(scc *securityapi.SecurityContextConstraints, annotations map[string]string) {
```

Comment:
@stlaz Do we know who and how will migrate sysctl annotations into the fields? Could you ensure that our release notes at least contain steps for migration? Otherwise, it's possible that users will have SCCs with annotations and we'll produce incorrect PSPs.

Another option, it to keep this code and support fields in addition to the annotations.

Reply:
I'll check that and the docs. It's possible that we may need to keep the code.